packages/medusa/src/scripts/gift-card-tax-rate-migration.ts

```diff
@@ -0,0 +1,110 @@
+import dotenv from "dotenv"
+import { createConnection, IsNull } from "typeorm"
+import Logger from "../loaders/logger"
+import { GiftCard } from "../models/gift-card"
+
+dotenv.config()
+
+const BATCH_SIZE = 1000
+const migrationName = 'gift-card-tax-rate-migration'
+const typeormConfig = {
```

Comment:
suggestion(non blocking): Since all scripts are using the same config, should we think to move that into a separate db config file? probably not in that pr but just wanted to mention it.